user/pages/03.commerce2/02.developer-guide/09.pricing/05.displaying-prices/docs.md

```diff
@@ -0,0 +1,75 @@
+
+Drupal Commerce actually provides two different implementations for this Price field formatter. A basic implementation is provided by the Commerce Price module. If the Commerce Order module is installed, then that basic implementation is replaced by one provided by Order module that uses the *Price Calculator* service to calculate the price a purchasable entity would have if it was in an order.
+
+#### Calculated price formatter - Price module version
+This formatter uses the `ChainPriceResolver` service to resolve the value of the price dynamically. It uses a quantity of 1, the current user, and the current store as the context for the price resolver. See the [Price resolvers](../price-resolvers) documentation to learn how Price resolvers calculate price values. Once the value is calculated, the formatter uses the [Currency formatter service](../formatting-prices) to render the Price based on its currency definition. 
+
+#### Calculated price formatter - Order module version
+The Order module version Price formatter provides all the functionality of the Price module version plus configuration options for applying adjustments. The field formatter can be configured to include any of the adjustment types defined for the site. If no adjustments are selected, the Order module version of the Calcualated price formatter behaves identically to Price module version.
+
+![Calculated price formatter configuration](../images/displaying-prices-2.png)
+
+The Calculated price formatter uses the `PriceCalculator` service provided by the Order module to calculate the value of the Price that's displayed. If any adjustment types are selected, an Order will be created (but not saved) that has a single Order item. The Order item has a quantity of 1 and the resolved price for its Unit price. Then all Order processors for the selected adjustment types *process* the Order, adding [Adjustments](../adjustments) when applicable. The final *adjusted total price* for the Order item is returned to the Calculated price formatter, where it is rendered based on its currency definition.
+
+## Rendering prices in Twig.
+
+The Commerce Price module also provides the `commerce_price_format` Twig filter which renders a given Price value object based on its currency definition. Here is an example of using the Twig filter by utilizing the `inline_template` element type.
+
+```
+$element['price'] = [
+  '#type' => 'inline_template',
+  '#template' => '{{ price|commerce_price_format }}',
+  '#context' => [
+    'price' => new \Drupal\commerce_price\Price('10.25', 'USD'),
+  ],
+];
+```
+
+This would render as $10.25.
+
+This Twig filter can be used on both [Price value objects](../prices) and arrays with `number` and `currency_code` keys. It uses the [Currency formatter service](../formatting-prices) to render the Price based on its currency definition.
```

Comment:
I think this is missing an example of formatting a price without twig. For example, there was a thread in Slack fairly recently about how it's not necessarily obvious how to do it https://app.slack.com/client/T06GX3JTS/C1TLCCF9B/thread/C1TLCCF9B-1585940275.355900

![image](https://user-images.githubusercontent.com/3623162/79598886-8355a080-8099-11ea-8c38-a5bc820b2f16.png)

Reply:
Thank you for the review and very helpful feedback!